### PR TITLE
Render AMP (text only)

### PR DESCRIPTION
**Pull request by iefserge**
Using Deku to render AMP document.

R = @kesla 
T = major

### CODE REVIEW

File: lib/render-article.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { element } from 'deku';
+import renderParagraph from './render-paragraph';
+
+export default function (tree) {
+  return tree
+    .filter(node => node.type === 'paragraph')
+    .map(renderParagraph);
+}
```
Comment (kesla) [inline lib/render-article.js]: couldn't this be a component? So something like `<Article data={tree}/>`

File: lib/components/article.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { element } from 'deku';
+import Paragraph from './paragraph';
+
+function renderItem (item) {
+  if (item.type === 'paragraph') {
+    return <Paragraph items={item.children || []} />;
```
Comment (kesla) [inline lib/components/article.js]: nitpick: I guess we can do `<Paragraph>{item.children}</Paragraph>` here?

Comment (kesla) [inline lib/components/article.js]: but no biggie, this is fine as well.

**Comment by kesla** [comment]
nice!

File: lib/components/article.js (hunk 2 above)
Comment (iefserge) [inline lib/components/article.js]: parser will always add children array (even if empty), right?

Comment (kesla) [inline lib/components/article.js]: Yeah

Comment (kesla) [inline lib/components/article.js]: Or rather, empty paragraphs are omitted.

**Pull request merged**